pkg/app/app.go

```diff
@@ -120,7 +120,7 @@ func RealMain(opts types.Options, d types.Deployer, tester types.Tester) (result
 		exec.InheritOutput(test)
 
 		envsForTester := os.Environ()
-		envsForTester = append(envsForTester, fmt.Sprintf("%s=%s", "ARTIFACTS", opts.ArtifactsDir()))
+		envsForTester = append(envsForTester, fmt.Sprintf("%s=%s", "ARTIFACTS", opts.RunDir()))
```

Comment:
let's also additionally expose this as `KUBETEST2_RUN_DIR`

Reply:
I like that - setting us up for the future cleanup (I like the 4 directories you suggested in your comment - just wanted to keep the PR focused).  Added!